roles/ubuntu-bootstrap/tasks/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+#  raw: cat /etc/issue.net | grep '{{ bootstrap_versions }}'
+
+- name: Bootstrap | Check if bootstrap is needed
+  raw: which python
+  register: need_bootstrap
+  ignore_errors: True
+
+- name: Bootstrap | Install python 2.x
+  raw: DEBIAN_FRONTEND=noninteractive apt install -y python-minimal
```

Comment:
apt works too, but it prints a warning that it is unsafe for scripting. Should I change it?